VERSION

```diff
@@ -1,4 +1,6 @@
-v9.2.2
+v9.2.3
```

Comment:
nit - https://semver.org/ With the new added functionality this should be a 9.3.0

Reply:
👍